NitroxPatcher/Patches/Player_Update_Patch.cs

```diff
@@ -24,12 +24,14 @@ public static void Postfix(Player __instance)
 
             foreach (KeyBinding keyBinding in keyBindingManager.KeyboardKeyBindings)
             {
-                bool isButtonDown = (bool)ReflectionHelper.ReflectionCall<GameInput>(null, "GetButtonDown", new Type[] { typeof(GameInput.Button) }, true, true, keyBinding.Button);
+                //Throws errors in the output_log
```

Comment:
Again, if something is broken, **don't just remove it** (locally is fine, just don't include it in a PR). Either try to fix it, or forward it to another Nitrox contributor.

I don't see any exception in my log, can you post them? Though they keybinding option doesn't show up anymore in the keyboard menu either... Maybe some configuration is broken on my system.